File: swixUseCases/swix-iOS-objc/swix-iOS-objc/ViewController.m
//
//  ViewController.m
//  swix-iOS-objc
//
//  Created by Scott Sievert on 7/12/14.
//  Copyright (c) 2014 com.scott. All rights reserved.
//

#import "ViewController.h"
#import "swixiOSobjc-Swift.h"

@interface ViewController ()

@end

@implementation ViewController
            
- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    test* t = [[test alloc] init];
    [t testFunc];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swixUseCases/swix-iOS/swix-iOS/ViewController.swift
//
//  ViewController.swift
//  swix-iOS
//
//  Created by Scott Sievert on 8/13/14.
//  Copyright (c) 2014 com.scott. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        var x = ones((4,4))
        var y = ones((4,4)) * 0.1
        var z = x *! y
        println(z)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

